bin/deploy

```diff
@@ -74,6 +75,28 @@ test -f "$LSSTSW/lfs/.git.deployed" || ( # git
     touch "$LSSTSW/lfs/.git.deployed"
 )
 
+test -f "${LSSTSW}/lfs/.git-lfs.deployed" || (
```

Comment:
You are mixing your ${LSSTSW} and $LSSTSW formatted bash variables. Semantically they are the same unless there are issues with tokenizing. But it still sets off my OCD or mental linter. :)